### DISCUSSION TITLE
iOS pretender app

### DISCUSSION THREAD

**Discussion by MKrabs** [TeamNewPipe/NewPipe#10829] [Questions] [answered]
Hello Newpipe team,

There is a rip-off, clone, great pretender app on the Apple App Store called "NewPipe", with the same logo and all.

https://apps.apple.com/de/app/newpipe-music-videos/id6450053445

Thought I'd let the team know, so you can take action.

**Comment by AudricV** [comment, marked as answer]
Already known, see https://github.com/TeamNewPipe/NewPipe/issues/539#issuecomment-1863011126. 
Please search (better, with different keywords?) for existing discussions and issues the next time you want to open a new discussion. Thank you.

**Reply by opusforlife2** [reply]
It's quite annoying that discussions aren't searchable in the same search bar like issues and PRs are. GH should have added an `is:discussion` filter.

**Reply by MKrabs** [reply]
Hi, thanks for the reply.
Yeah I tried to search for a bit, but searching for discussions is not perfect.
Good to hear that you've already had this reported.

I was quite angered when a "Pay for premium" Pop-up came.
